numpy/random/_generator.pyx

```diff
@@ -776,13 +776,13 @@ cdef class Generator:
         else:
             if size > pop_size:
                 raise ValueError("Cannot take a larger sample than "
-                                 "population when 'replace=False'")
+                                 "population when replace is False")
```

Comment:
If using backticks for parameter names, don’t forget them for _replace_.

Reply:
I removed all backticks since these appear to be non standard in this module (I couldn't find any other uses).